src/features/team-load-channel.js

```diff
@@ -0,0 +1,156 @@
+                    tweetJS = $("<script>", {"src":"http://platform.twitter.com/widgets.js", "type":"text/javascript"});
+                tweetBtn.text("Tweet");
+                $("#twitter_share_button").append(tweetBtn, tweetJS);
+
+                var chanLinkSpan = $("<strong>"),
+                    chanLink = $("<a>", {"id":"live_channel_name", "href":"/"+chan});
+                chanLink.text(jsnTeam[i].channel.display_name);
+                chanLinkSpan.append(chanLink);
+
+                var viewerCount = $("<span>", {"id":"channel_viewer_count", "class":"stat"});
+                viewerCount.text(jsnTeam[i].channel.current_viewers);
+
+                var viewsCount = $("<span>", {"id":"views_count", "class":"stat"});
+                viewsCount.text(jsnTeam[i].channel.total_views+" ");
+
+                var followersCount = $("<span>", {"id":"followers_count", "class":"stat"});
+                followersCount.text(jsnTeam[i].channel.followers_count);
+
+                var descHolder = $("<div>", {"id":"description"});
+                descHolder.html("<b>Channel Description:</b><br>"+jsnTeam[i].channel.description+"<br><br><b>Broadcast Title:</b><br>"+jsnTeam[i].channel.title);
+
+                $("#stats_and_description").empty();
+                var chanActions = $("<div>", {"id":"channel_actions"}),
+                    followBtn = $("<a>", {"id":"followbtn", "data-ember-action":"135", "class":"js-follow follow button primary action"});
+                followBtn.text(" Follow ");
+                
+                followBtn.click(function(e) {
+                    followcurrentchannel();
+                });
+                chanActions.append(followBtn);
+
+                var shareBtn   = $("<div>", {"id":"sharebtn", "class":"button action primary"});
+                shareBtn.text("Share");
+
+                if(val != 0) {
+                    var subBtn = $("<a>", {"id":"subscribe_action", "class":"action button js-sub-button primary subscribe-button", "href":"/"+chan+"/subscribe?ref=below_video_subscribe_button", "target":"_blank"}),
+                        subTxt = $("<span>", {"class":"subscribe-text"}),
+                        subPrice = $("<span>", {"class":"subscribe-price"});
+                        
+                    subTxt.text("Subscribe");
+                    subPrice.text(val);
+                    
+                    subBtn.append(subTxt, subPrice);
+                    chanActions.append(subBtn, shareBtn);
+                } else {
+                    chanActions.append(shareBtn);
+                }
+
+                $("#stats_and_description").append(chanLinkSpan, " playing ", jsnTeam[i].channel.meta_game, "<br>", viewerCount, viewsCount, followersCount, chanActions, descHolder);
```

Comment:
Holy crap, I'm so sorry for causing you to spend this much time holding my hand. But again, I would like to thank you for your patience in explaining all of this to me.

I will work on this rather long list (which I'm sure will grown longer hehe. sorry!) over the next few days, and get a new commit up ASAP.